CMakeLists.txt

```diff
@@ -104,6 +109,7 @@ set(STLINK_HEADERS
         src/stlink-lib/md5.h
         src/stlink-lib/sg.h
         src/stlink-lib/usb.h
+		src/stlink-lib/helper.h
```

Comment:
Please correct the indentation in this file to match the current formatting.
Don't use tabspacing as they vary between architectures.
(I believe this is the reason for the misalignment.)

Reply:
This is where I really overlooked. I'll fix it soon.

Reply:
Solved by additional commit